### PR TITLE
Adjust CODEOWNERS for .github/

### PR DESCRIPTION
**Pull request by aryx**
test plan:
wait for green CI Checks

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
 # Matches all files in the .github/ dir, recursively.
-.github/ @spencerdrak @cgdolan @aryx @bmahe @apt-itude
+.github/ @aryx
```
Comment (nmote) [inline .github/CODEOWNERS:3]: Why not just remove them altogether?

**Review by nmote — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (aryx) [reply .github/CODEOWNERS:3]: I still want to review those. I do review them.

Comment (emjin) [reply .github/CODEOWNERS:3]: @aryx Is there a way for you to just set up an alert? I'm not a fan of having a single codeowner ever; when that person is out then everyone is blocked.

Comment (aryx) [reply .github/CODEOWNERS:3]: I can remove the file

**Pull request merged**